vxdev/run-vxadmin.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Version=1.0
+Name=VxAdmin
+Icon=votingworks-logo.png
+Exec=gnome-terminal -- /bin/sh -c 'env PATH=$PATH:/sbin /vx/code/vxsuite-complete-system/run.sh election-manager'
```

Comment:
is the `gnome-terminal` call needed here? I would think just `/bin/sh -c` or `/bin/bash -c` would suffice.

Reply:
I was having issues getting the desktop icon to work without launching it from a terminal, and the terminal was nice for debugging when there were errors but I will try again without this and make sure its necessary.

Reply:
oh, one more thought: setting the PATH should also be unnecessary, since it's globally set by the setup script.

Reply:
Yes, I don't know why I have to do this but for some reason I do when it launches from the desktop icon it doesn't have sbin anymore 🤷‍♀️

Reply:
huh, weird. I wonder if that would be true after a reboot?

Reply:
I wonder if this has to do with where you're setting your `$PATH` environment variable. Is it in your `.bash_profile` ? Somewhere else?